### PR TITLE
Changing headers to sentence case #5

### PR DESCRIPTION
**Pull request by apinnick**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

**Review by bangelic — COMMENTED** [review]
Changes LGTM from a docs standpoint.

File: guides/common/modules/snip_host-names-for-http-proxy.adoc [hunk 1]
```diff
@@ -12,3 +12,7 @@ ifdef::satellite[]
 | cert-api.access.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
 endif::[]
 |====
+
+ifdef::orcharhino[]
```
Comment (maximiliankolb) [inline guides/common/modules/snip_host-names-for-http-proxy.adoc:16]: This diff should not be part of your PR. Can you please rebase? Then I'll merge.

File: guides/common/modules/ref_provisioning-template-pxelinux-discovery-snippet.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [id="Provisioning_Template_PXELinux_Discovery_Snippet_{context}"]
-= Provisioning Template PXELinux Discovery Snippet
+= Provisioning template PXELinux discovery snippet
```
Comment (maximiliankolb) [inline guides/common/modules/ref_provisioning-template-pxelinux-discovery-snippet.adoc:2]:
```suggestion
= Provisioning template PXELinux Discovery snippet
```

File: guides/common/modules/ref_troubleshooting-discovery.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [id="Troubleshooting_Discovery_{context}"]
-= Troubleshooting Discovery
+= Troubleshooting discovery
```
Comment (maximiliankolb) [inline guides/common/modules/ref_troubleshooting-discovery.adoc:2]:
```suggestion
= Troubleshooting Discovery
```

I think this refers to "Foreman Discovery", a plug-in.

Comment (apinnick) [reply guides/common/modules/ref_troubleshooting-discovery.adoc:2]: Looks like it refers to PXE discovery:

> Check the PXELinux template in use and determine the PXE discovery snippet it includes

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked to 3.9: :heavy_check_mark: 
   8e0027f47..5786c16dc  3.9 -> 3.9

**Pull request merged**